runtime/development/Cargo.toml

```diff
@@ -189,6 +190,7 @@ std = [
   "pallet-collective/std",
   "pallet-connectors/std",
   "pallet-connectors-gateway/std",
+  "pallet-connectors-gateway/std",
```

Comment:
This is duplicated (probably some git issue)

Reply:
Weird, thanks for catching this!